greengrass_shadow_delete_block.py

```diff
@@ -0,0 +1,36 @@
+import json
+
+from nio.properties import (VersionProperty, Property)
+from nio.signal.base import Signal
+from nio.util.discovery import discoverable
+
+from .greengrass_shadow_base_block import GreenGrassMQTTShadowBase
+
+
+@discoverable
+class GreenGrassShadowDelete(GreenGrassMQTTShadowBase):
+    """Delete device shadow properties on the local MQTT system"""
+
+    version = VersionProperty('0.1.0')
+    data_to_delete = Property(title="Data to delete",
+                              default="{{ $.to_dict() }}",
+                              allow_none=False)
+
+    def _delete_callback(self, payload, responseStatus, token):
```

Comment:
I'm not sure what a 'token' is in this context, but is there potential security concerns with logging it?